### ISSUE TITLE
make run failed on centos:  CXX(target) Release/obj.target/bufferutil/src/bufferutil.o

### ISSUE THREAD

**Issue by gclsoft** [Automattic/wp-calypso#594]
```
> bufferutil@1.2.1 install /root/work/wp-calypso/node_modules/socket.io/node_modules/engine.io/node_modules/ws/node_modules/bufferutil
> prebuild --download

prebuild http request GET https://github.com/bitpay/bufferutil/releases/download/v1.2.1/bufferutil-v1.2.1-node-v11-linux-x64.tar.gz
prebuild http 404 https://github.com/bitpay/bufferutil/releases/download/v1.2.1/bufferutil-v1.2.1-node-v11-linux-x64.tar.gz
prebuild WARN install Failed to download prebuild 
make[1]: Entering directory `/root/work/wp-calypso/node_modules/socket.io/node_modules/engine.io/node_modules/ws/node_modules/bufferutil/build' 
  CXX(target) Release/obj.target/bufferutil/src/bufferutil.o
  SOLINK_MODULE(target) Release/obj.target/bufferutil.node
  COPY Release/bufferutil.node
make[1]: Leaving directory `/root/work/wp-calypso/node_modules/socket.io/node_modules/engine.io/node_modules/ws/node_modules/bufferutil/build' 
Binary downloaded and installed at /root/work/wp-calypso/node_modules/node-sass/vendor/linux-x64-11/binding.node
 
> node-sass@3.3.3 postinstall /root/work/wp-calypso/node_modules/node-sass
> node scripts/build.js
 
` /root/work/wp-calypso/node_modules/node-sass/vendor/linux-x64-11/binding.node ` exists. 
 testing binary. 
Binary is fine; exiting.
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /root/work/wp-calypso/npm-debug.log
npm ERR! not ok code 0
make: *** [node_modules] Error 1
```

**Comment by nb** [comment]
I don’t really see what is the error here, `bufferutil` seems to be installed correctly. @gclsoft, could you post the whole `/root/work/wp-calypso/npm-debug.log` in a gist, please?

**Comment by gclsoft** [comment]
Thanks @nb log https://gist.github.com/gclsoft/e362377fdac6437fcccd  :
```
3257 79758 silly lockFile 9c19197c-wrappy-1 wrappy@1                                                                                                      
3258 79759 silly lockFile 0340bce5-filename-regex-2-0-0 filename-regex@2.0.0                                                                              
3259 79760 silly lockFile 0340bce5-filename-regex-2-0-0 filename-regex@2.0.0                                                                              
3260 79761 silly lockFile 1f243756-kind-of-2-0-1 kind-of@2.0.1                                                                                            
3261 79762 silly lockFile 1f243756-kind-of-2-0-1 kind-of@2.0.1                                                                                            
3262 79763 silly addNameRange number 2 { name: 'json-stringify-safe',                                                                                     
3263 79763 silly addNameRange   range: '>=5.0.0-0 <5.1.0-0',                                                                                              
3264 79763 silly addNameRange   hasData: true }                                                                                                           
3265 79764 silly addNameRange versions [ 'json-stringify-safe',                                                                                           
3266 79764 silly addNameRange   [ '2.0.0', '3.0.0', '4.0.0', '5.0.0', '5.0.1' ] ]                                                                         
3267 79765 verbose addNamed [ 'json-stringify-safe', '5.0.1' ]                                                                                            
3268 79766 verbose addNamed [ '5.0.1', '5.0.1' ]                                                                                                          
3269 79767 silly lockFile 4de81deb-json-stringify-safe-5-0-1 json-stringify-safe@5.0.1                                                                    
3270 79768 verbose lock json-stringify-safe@5.0.1 /root/.npm/4de81deb-json-stringify-safe-5-0-1.lock                                                      
3271 79769 silly gunzTarPerm modes [ '755', '644' ]                                                                                                       
3272 79770 silly addNameRange { name: 'ieee754', range: '>=1.1.4-0 <2.0.0-0', hasData: false }                                                            
3273 79771 silly addNameRange { name: 'is-array', range: '>=1.0.1-0 <2.0.0-0', hasData: false }                                                           
3274 79772 verbose readDependencies using package.json deps                                                                                               
3275 79773 verbose readDependencies using package.json deps                                                                                               
3276 79774 silly gunzTarPerm extractEntry svg/1f502.svg                                                                                                   
3277 79775 silly gunzTarPerm extractEntry svg/1f503.svg                                                                                                   
3278 79776 silly gunzTarPerm extractEntry library/modules/es6.function.has-instance.js                                                                    
3279 79777 silly gunzTarPerm extractEntry library/modules/es6.function.name.js                                                                            
3280 79778 silly lockFile d67f0508--optionator-node-modules-deep-is tar:///root/work/wp-calypso/node_modules/eslint/node_modules/optionator/node_modules/d     eep-is                                                                                                                                               
3281 79779 silly lockFile d67f0508--optionator-node-modules-deep-is tar:///root/work/wp-calypso/node_modules/eslint/node_modules/optionator/node_modules/d     eep-is                                                                                                                                               
3282 79780 verbose readDependencies using package.json deps                                                                                               
3283 79781 silly gunzTarPerm extractEntry .jshintrc                                                                                                       
3284 79782 silly gunzTarPerm extractEntry Makefile                                                                                                        
3285 79783 verbose tar unpack /root/.npm/once/1.3.3/package.tgz                                                                                           
3286 79784 silly lockFile 43bf7cec--end-of-stream-node-modules-once tar:///root/work/wp-calypso/node_modules/socket.io/node_modules/socket.io-client/node_     modules/engine.io-client/node_modules/ws/node_modules/bufferutil/node_modules/prebuild/node_modules/tar-stream/node_modules/end-of-stream/node_module     s/once                                                                                                                                               
3287 79785 verbose lock tar:///root/work/wp-calypso/node_modules/socket.io/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws/nod     e_modules/bufferutil/node_modules/prebuild/node_modules/tar-stream/node_modules/end-of-stream/node_modules/once /root/.npm/43bf7cec--end-of-stream-no     de-modules-once.lock                                                                                                                                 
3288 79786 silly lockFile f6df5bc1--root-npm-once-1-3-3-package-tgz tar:///root/.npm/once/1.3.3/package.tgz                                               
3289 79787 verbose lock tar:///root/.npm/once/1.3.3/package.tgz /root/.npm/f6df5bc1--root-npm-once-1-3-3-package-tgz.lock                                 
3290 79788 silly resolved [ { name: 'wrappy',                                                                                                             
3291 79788 silly resolved     version: '1.0.1',                                                                                                           
3292 79788 silly resolved     description: 'Callback wrapping utility',  
3293 79788 silly resolved     main: 'wrappy.js',                                                                                                          
3294 79788 silly resolved     directories: { test: 'test' },                                                                                              
3295 79788 silly resolved     dependencies: {},                                                                                                           
3296 79788 silly resolved     devDependencies: { tap: '^0.4.12' },                                                                                        
3297 79788 silly resolved     scripts: { test: 'tap test/*.js' },                                                                                         
3298 79788 silly resolved     repository: { type: 'git', url: 'git+https://github.com/npm/wrappy.git' },                                                  
3299 79788 silly resolved     author:                                                                                                                     
3300 79788 silly resolved      { name: 'Isaac Z. Schlueter',                                                                                              
3301 79788 silly resolved        email: 'i@izs.me',                                                                                                       
3302 79788 silly resolved        url: 'http://blog.izs.me/' },                                                                                            
3303 79788 silly resolved     license: 'ISC',                                                                                                             
3304 79788 silly resolved     bugs: { url: 'https://github.com/npm/wrappy/issues' },                                                                      
3305 79788 silly resolved     homepage: 'https://github.com/npm/wrappy',                                                                                  
3306 79788 silly resolved     readme: '# wrappy\n\nCallback wrapping utility\n\n## USAGE\n\n```javascript\nvar wrappy = require("wrappy")\n\n// var wrappe     r = wrappy(wrapperFunction)\n\n// make sure a cb is called only once\n// See also: http://npm.im/once for this specific use case\nvar once = wrappy(f     unction (cb) {\n  var called = false\n  return function () {\n    if (called) return\n    called = true\n    return cb.apply(this, arguments)\n  }\n}     )\n\nfunction printBoo () {\n  console.log(\'boo\')\n}\n// has some rando property\nprintBoo.iAmBooPrinter = true\n\nvar onlyPrintOnce = once(printBo     o)\n\nonlyPrintOnce() // prints \'boo\'\nonlyPrintOnce() // does nothing\n\n// random property is retained!\nassert.equal(onlyPrintOnce.iAmBooPrinter     , true)\n```\n',                                                                                                                                     
3307 79788 silly resolved     readmeFilename: 'README.md',                                                                                                
3308 79788 silly resolved     _id: 'wrappy@1.0.1',                                                                                                        
3309 79788 silly resolved     _from: 'wrappy@1' } ]                                                                                                       
3310 79789 info install wrappy@1.0.1 into /root/work/wp-calypso/node_modules/socket.io/node_modules/socket.io-client/node_modules/engine.io-client/node_mo     dules/ws/node_modules/bufferutil/node_modules/prebuild/node_modules/pump/node_modules/once                                                           
3311 79790 info installOne wrappy@1.0.1                                                                                                                   
3312 79791 silly lockFile bdf1559f-filename-regex-2-0-0 filename-regex@^2.0.0                                                                             
3313 79792 silly lockFile bdf1559f-filename-regex-2-0-0 filename-regex@^2.0.0                                                                             
3314 79793 silly lockFile 19d82d7d-kind-of-2-0-1 kind-of@^2.0.1                                                                                           
3315 79794 silly lockFile 19d82d7d-kind-of-2-0-1 kind-of@^2.0.1                                                                                           
3316 79795 silly gunzTarPerm modes [ '755', '644' ]                                                                                                       
3317 79796 verbose cache add [ 'once@~1.3.0', null ]                                                                                                      
3318 79797 verbose cache add name=undefined spec="once@~1.3.0" args=["once@~1.3.0",null]                                                                  
3319 79798 verbose parsed url { protocol: null,                                                                                                           
3320 79798 verbose parsed url   slashes: null,                                                                                                            
3321 79798 verbose parsed url   auth: null,                                                                                                               
3322 79798 verbose parsed url   host: null,                                                                                                               
3323 79798 verbose parsed url   port: null,                                                                                                               
3324 79798 verbose parsed url   hostname: null,                                                                                                           
3325 79798 verbose parsed url   hash: null,                                                                                                               
3326 79798 verbose parsed url   search: null,                                                                                                             
3327 79798 verbose parsed url   query: null,                                                                                                              
3328 79798 verbose parsed url   pathname: 'once@~1.3.0', 
3329 79798 verbose parsed url   path: 'once@~1.3.0',                                                                                                      
3330 79798 verbose parsed url   href: 'once@~1.3.0' }                                                                                                     
3331 79799 verbose cache add name="once" spec="~1.3.0" args=["once","~1.3.0"]                                                                             
3332 79800 verbose parsed url { protocol: null,                                                                                                           
3333 79800 verbose parsed url   slashes: null,                                                                                                            
3334 79800 verbose parsed url   auth: null,                                                                                                               
3335 79800 verbose parsed url   host: null,                                                                                                               
3336 79800 verbose parsed url   port: null,                                                                                                               
3337 79800 verbose parsed url   hostname: null,                                                                                                           
3338 79800 verbose parsed url   hash: null,                                                                                                               
3339 79800 verbose parsed url   search: null,                                                                                                             
3340 79800 verbose parsed url   query: null,                                                                                                              
3341 79800 verbose parsed url   pathname: '~1.3.0',                                                                                                       
3342 79800 verbose parsed url   path: '~1.3.0',                                                                                                           
3343 79800 verbose parsed url   href: '~1.3.0' }                                                                                                          
3344 79801 verbose addNamed [ 'once', '~1.3.0' ]                                                                                                          
3345 79802 verbose addNamed [ null, '>=1.3.0-0 <1.4.0-0' ]                                                                                                
3346 79803 silly lockFile 078063e6-once-1-3-0 once@~1.3.0                                                                                                 
3347 79804 verbose lock once@~1.3.0 /root/.npm/078063e6-once-1-3-0.lock                                                                                   
3348 79805 silly gunzTarPerm extractEntry LICENSE                                                                                                         
3349 79806 silly gunzTarPerm extractEntry jsonist.js                                                                                                      
3350 79807 info postinstall lodash._baseassign@3.2.0                                                                                                      
3351 79808 verbose readDependencies using package.json deps                                                                                               
3352 79809 silly resolved []                                                                                                                              
3353 79810 verbose about to build /root/work/wp-calypso/node_modules/socket.io/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws     /node_modules/utf-8-validate/node_modules/prebuild/node_modules/tar-stream/node_modules/end-of-stream/node_modules/once/node_modules/wrappy          
3354 79811 info build /root/work/wp-calypso/node_modules/socket.io/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws/node_module     s/utf-8-validate/node_modules/prebuild/node_modules/tar-stream/node_modules/end-of-stream/node_modules/once/node_modules/wrappy                      
3355 79812 verbose linkStuff [ false,                                                                                                                     
3356 79812 verbose linkStuff   false,                                                                                                                     
3357 79812 verbose linkStuff   false,                                                                                                                     
3358 79812 verbose linkStuff   '/root/work/wp-calypso/node_modules/socket.io/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws/n     ode_modules/utf-8-validate/node_modules/prebuild/node_modules/tar-stream/node_modules/end-of-stream/node_modules/once/node_modules' ]                
3359 79813 info linkStuff wrappy@1.0.1                                                                                                                    
3360 79814 verbose linkBins wrappy@1.0.1                                                                                                                  
3361 79815 verbose linkMans wrappy@1.0.1                                                                                                                  
3362 79816 verbose rebuildBundles wrappy@1.0.1                                                                                                            
3363 79817 silly lockFile 4b81c690-ream-node-modules-string-decoder tar:///root/work/wp-calypso/node_modules/socket.io/node_modules/engine.io/node_modules     /ws/node_modules/bufferutil/node_modules/prebuild/node_modules/ghreleases/node_modules/ghutils/node_modules/jsonist/node_modules/hyperquest/node_modu     les/duplexer2/node_modules/readable-stream/node_modules/string_decoder 
3364 79818 silly lockFile 4b81c690-ream-node-modules-string-decoder tar:///root/work/wp-calypso/node_modules/socket.io/node_modules/engine.io/node_modules     /ws/node_modules/bufferutil/node_modules/prebuild/node_modules/ghreleases/node_modules/ghutils/node_modules/jsonist/node_modules/hyperquest/node_modu     les/duplexer2/node_modules/readable-stream/node_modules/string_decoder                                                                               
3365 79819 verbose readDependencies using package.json deps                                                                                               
3366 79820 silly resolved []                                                                                                                              
3367 79821 verbose about to build /root/work/wp-calypso/node_modules/socket.io/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws     /node_modules/bufferutil/node_modules/prebuild/node_modules/npmlog/node_modules/ansi                                                                 
3368 79822 info build /root/work/wp-calypso/node_modules/socket.io/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws/node_module     s/bufferutil/node_modules/prebuild/node_modules/npmlog/node_modules/ansi                                                                             
3369 79823 verbose linkStuff [ false,                                                                                                                     
3370 79823 verbose linkStuff   false,                                                                                                                     
3371 79823 verbose linkStuff   false,                                                                                                                     
3372 79823 verbose linkStuff   '/root/work/wp-calypso/node_modules/socket.io/node_modules/socket.io-client/node_modules/engine.io-client/node_modules/ws/n     ode_modules/bufferutil/node_modules/prebuild/node_modules/npmlog/node_modules' ]                                                                     
3373 79824 info linkStuff ansi@0.3.0                                                                                                                      
3374 79825 verbose linkBins ansi@0.3.0                                                                                                                    
3375 79826 verbose linkMans ansi@0.3.0                                                                                                                    
3376 79827 verbose rebuildBundles ansi@0.3.0                                                                                                              
3377 79828 silly lockFile 2088c8b3-he-node-modules-lodash-getnative tar:///root/work/wp-calypso/node_modules/eslint/node_modules/lodash.omit/node_modules/     lodash._basedifference/node_modules/lodash._createcache/node_modules/lodash._getnative                                                               
3378 79829 silly lockFile 2088c8b3-he-node-modules-lodash-getnative tar:///root/work/wp-calypso/node_modules/eslint/node_modules/lodash.omit/node_modules/     lodash._basedifference/node_modules/lodash._createcache/node_modules/lodash._getnative                                                               
3379 79830 info preinstall source-map@0.4.4                                                                                                               
3380 79831 error Error: ENOENT, lstat '/root/tmp/npm-15709-TB12kzi3/1448337472649-0.7886159582994878/package/test/test.js'                                
3381 79832 error If you need help, you may report this log at:                                                                                            
3382 79832 error     <http://github.com/isaacs/npm/issues>                                                                                                
3383 79832 error or email it to:                                                                                                                          
3384 79832 error     <npm-@googlegroups.com>                                                                                                              
3385 79833 error System Linux 4.1.5-x86_64-linode61                                                                                                       
3386 79834 error command "node" "/usr/bin/npm" "install"                                                                                                  
3387 79835 error cwd /root/work/wp-calypso                                                                                                                
3388 79836 error node -v v0.10.33                                                                                                                         
3389 79837 error npm -v 1.3.6                                                                                                                             
3390 79838 error path /root/tmp/npm-15709-TB12kzi3/1448337472649-0.7886159582994878/package/test/test.js                                                  
3391 79839 error fstream_path /root/tmp/npm-15709-TB12kzi3/1448337472649-0.7886159582994878/package/test/test.js                                          
3392 79840 error fstream_type File                                                                                                                        
3393 79841 error fstream_class FileWriter                                                                                                                 
3394 79842 error code ENOENT                                                                                                                              
3395 79843 error errno 34                                                                                                                                 
3396 79844 error fstream_stack /usr/lib/node_modules/fstream/lib/writer.js:284:26                                                                         
3397 79844 error fstream_stack Object.oncomplete (fs.js:107:15)                                                                                           
3398 79845 verbose exit [ 34, true ]         
```

**Comment by Wodric** [comment]
Hi, i got some similar error during installing. I'm using linux mint 17.2. I installed npm and node from official repository. Id i remember well i got npm version 1.3.10. 

Firstly if i just clone and execute "make run" I got an error make: *** [nodes_module] error 1 .
After this i tried to install nodes module manually (npm install) and saw lot on install and somme error on it.
See it on the following file :
[npm_install_log.txt](https://github.com/Automattic/wp-calypso/files/43216/npm_install_log.txt)
